crates/core/src/errors.rs

```diff
@@ -134,6 +134,13 @@ pub enum DownloadError {
     DependencyNotFound(String),
 }
 
+#[derive(Error, Debug)]
+#[non_exhaustive]
+pub enum InitError {
+    #[error("no config file specified for --config_location")]
```

Comment:
Clap automatically converts underscore to hyphen so it should be `--config-location`

Reply:
fixed